### DISCUSSION TITLE
Tracking jacktrip in downstream package ecosystem repositories

### DISCUSSION THREAD

**Discussion by luzpaz** [jacktrip/jacktrip#1204] [General]
The purpose of this ticket is to track the stable versions downstream (and make requested version bumps when needed): 

<a href="https://repology.org/project/jacktrip/versions">
    <img src="https://repology.org/badge/vertical-allrepos/jacktrip.svg?header=jacktrip&columns=3" alt="Packaging status">
</a>

- [ ] Alpine
- [ ] Arch (maintainer hasn't responded) <-- **Any volunteers?**
- [ ] chocolatey (https://github.com/chocolatey-community/chocolatey-package-requests/issues/1444)
- [ ] homebrew cask
- [ ] nixOS (https://github.com/NixOS/nixpkgs/issues/247933)
- [ ] wikipedia - create a dedicated page for the jacktrip project (this auto-creates a [wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page) page) <-- **Any volunteers?**
- [ ] winget (https://github.com/microsoft/winget-pkgs/issues/115725)

**Comment by ntonnaett** [comment]
That's a nice overview. Thank you! Could we remove EOL versions like all the the red Fedoras? There's also the ppa similar to Debian backports that we have in our documentation: ppa:umlaeute/jacktrip.

**Comment by luzpaz** [comment]
> Could we remove EOL versions like all the the red Fedoras?  

Not sure repology badge has this ability (yet). 

> ppa

I don't think repology supports showing PPAs, Flatpaks, Snaps. It will show AppImages from time to time.

**Comment by luzpaz** [comment]
Problem with nixpkgs: https://github.com/NixOS/nixpkgs/issues/247933#issuecomment-1685255300

**Comment by umlaeute** [comment]
sidenote: wouldn't this be in better hands as part of the [discussions](https://github.com/jacktrip/jacktrip/discussions) (it appears to not really be an "issue" (as in bug/feature-request/...)

**Comment by luzpaz** [comment]
A user with the right permissions can easily migrate this to a discussion (github has an automagic option for that)

**Comment by luzpaz** [comment]
Can a dev take a look at https://github.com/NixOS/nixpkgs/issues/247933#issuecomment-1685255300 please?

**Reply by ntonnaett** [reply]
Done

**Comment by luzpaz** [comment]
Looks like Arch has an [extra-staging repo](https://archlinux.org/packages/extra-staging/x86_64/jacktrip/) where 2.0.2 is being distributed from. I'll try to let repology know about that.
